services/resource-service/server/pom.xml

```diff
@@ -47,7 +47,7 @@
         <dependency>
             <groupId>org.apache.airavata</groupId>
             <artifactId>registry-api-stubs</artifactId>
-            <version>0.19-SNAPSHOT</version>
```

Comment:
Bring 0.19-SNAPSHOT back. You should build latest mft-integration branch of airavata repo locally to get this dependency version

Reply:
The highest version available in the maven repository was .17 , hence we changed this. 
Could you give us further input on how to build .19?

Reply:
You can clone airavata repository https://github.com/apache/airavata and checkout the mft-integration branch. Run a mvn install on that and it will generate 0.19-SNAPSHOTS